### PR TITLE
hello git

### PR DESCRIPTION
**Pull request by cward-extron**
hello world

### CODE REVIEW

File: _posts/0000-01-02-cward-extron.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (nvinchhi-extron) [inline _posts/0000-01-02-cward-extron.md:1]: This is a comment

**Comment by cmccurdy-extron** [comment]
Heya chris!

File: _posts/0000-01-02-cward-extron.md (hunk 1 above)
Comment (cmccurdy-extron) [reply _posts/0000-01-02-cward-extron.md:1]: image

Comment (kbeelman-extron) [reply _posts/0000-01-02-cward-extron.md:1]: animated praise

**Review by cmccurdy-extron — APPROVED** [review]

**Review by cward-extron — COMMENTED** [review]
lgtm

**Review by nvinchhi-extron — APPROVED** [review]
text crow of praise

File: _posts/0000-01-02-cward-extron.md (hunk 1 above)
Comment (cward-extron) [reply _posts/0000-01-02-cward-extron.md:1]: hardy har har

File: _posts/0000-01-02-cward-extron.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+layout: slide
+title: "Welcome to our second slide!"
+---
```
Comment (cward-extron) [inline _posts/0000-01-02-cward-extron.md:4]: comment 2

File: _posts/0000-01-02-cward-extron.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+whatever works
```
Comment (cward-extron) [inline _posts/0000-01-02-cward-extron.md:5]: whatever

**Review by cward-extron — COMMENTED** [review]
lots of comments

**Pull request merged**